src/styles/globals.css

```diff
@@ -15,7 +15,6 @@ body {
 	--oex-lighter-grey: #e5e5e5;
 	--oex-lightest-grey: #f2f0f0;
 	--oex-orange: #fe7624;
-	--oex-light-orange: #fef9f5;
```

Comment:
some of these variables look alike, they need to be reduced

Reply:
I won't add anymore colors, i will use only these ones. I'm trying to get the design